### PR TITLE
Fixes #1805. Fixes Store call error messages

### LINKED ISSUE #1805: Change Rows call format to be more similar to other calls
### Description

Currently `Rows` call has this structure: 
```
Rows(field=<FIELD>, previous=<UINT|STRING>, limit=<UINT>, column=<UINT|STRING>)
```
This is different from other similar calls, which take a required field name, i.e.:
```
SetRowAttrs(<FIELD>, <ROW>,
            <ATTR_NAME=ATTR_VALUE>,
            [ATTR_NAME=ATTR_VALUE ...])

TopN(<FIELD>, [ROW_CALL], [n=UINT],
     [attrName=<ATTR_NAME>, attrValues=<[]ATTR_VALUE>])
```
It would be better to change the structure of the `Rows` call, so it is more like the `SetRowAttrs` and `TopN` calls, so it becomes:
```
Rows(<FIELD>, previous=<UINT|STRING>, limit=<UINT>, column=<UINT|STRING>)
```

### Success criteria (What criteria will consider this ticket closeable?)

`Rows` call is structured as:
```
Rows(<FIELD>, previous=<UINT|STRING>, limit=<UINT>, column=<UINT|STRING>)
```

### PR DESCRIPTION
**Pull request by yuce**
## Overview

Fixes #1805 

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

**Comment by yuce** [comment]
@jaffee  I've updated `Rows` to accept  the `field` param for backwards compat.

**Comment by travisturner** [comment]
I may be doing something wrong, but I get an error when I use the "backward compatible" method:
```
curl http://localhost:10101/index/records/query -d 'GroupBy(Rows(field=user))'
{"error":"executing: getting group by iterator for shard 0: Rows call must have field with valid (string) field name. Got \u003cnil\u003e of type \u003cnil\u003e"}
```
For reference, all of these work fine:
```
curl http://localhost:10101/index/records/query -d 'Rows(user)'
curl http://localhost:10101/index/records/query -d 'Rows(field=user)'
curl http://localhost:10101/index/records/query -d 'GroupBy(Rows(user))'
```

**Comment by yuce** [comment]
@travisturner updated and added a legacy `Groupby(Rows(field=...))` test.

File: Makefile [hunk 1]
```diff
@@ -139,7 +139,7 @@ gometalinter: require-gometalinter
 	    --enable=ineffassign \
 	    --enable=interfacer \
 	    --enable=maligned \
-	    --enable=megacheck \
+	    --enable=staticcheck \
```
Comment (codysoyland) [inline Makefile]: This might be pedantic, but could you move this down so the list of linters remains in alphabetical order?

Comment (yuce) [reply Makefile]: sure, updated

**Pull request merged**